tests/apps/utils/http-server.go

```diff
@@ -55,6 +61,19 @@ func StartServer(port int, appRouter func() *mux.Router, allowHTTP2 bool) {
 		}
 	}
 
+	var certFile, keyFile string
+	if enableTLS {
+		certFile, keyFile, err = getTLSCertAndKey()
+		if err != nil {
+			log.Printf("Failed to get TLS cert and key: %v, falling back to HTTP", err)
+			enableTLS = false
+		} else {
+			server.TLSConfig = &tls.Config{
+				ClientAuth: tls.RequestClientCert,
```

Comment:
I'm not sure I understand the goal of this. Are you requesting client certificates to do mTLS?

Reply:
I looked around the PR but I don't see any client sending TLS certs - am I missing something?

Reply:
I was playing with this parameter and I missed cleaning that up, removed it, thanks!